### ISSUE TITLE
INVALID_SESSION_ID

### ISSUE THREAD

**Issue by adal877** [sorenkrabbe/Chrome-Salesforce-inspector#228]
For about 2, or 3 days I've been getting the following error:
```
list entity definitions:  where QualifiedApiName >= 'M' limit 2000 SalesforceRestError: INVALID_SESSION_ID: Session expired or invalid  
    at Object.rest (inspector.js:75:17)  
    at async Promise.all (chrome-extension://aodjmnfhjibkcdimpodiifdjnnncaafh/index 3)  
```
Since then I've been trying to logout, clear all cookies, login again, uninstalling and installing again, and nothing happaned. Any idea how to solve it?

**Comment by dschach** [comment]
Are you using Enhanced Domain?

**Comment by adal877** [comment]
> Are you using Enhanced Domain?

No. Actually, I was using the inspector, with no issue, in the same org for a couple of months.

**Comment by DarkSkull91** [comment]
Same error here.

**Comment by DarkSkull91** [comment]
I noticed that invoking the extention with the enhanced domain it works: chrome-extension://aodjmnfhjibkcdimpodiifdjnnncaafh/data-export.html?host=myorg.sandbox.my.salesforce.com

**Comment by adal877** [comment]
> I noticed that invoking the extention with the enhanced domain it works: chrome-extension://aodjmnfhjibkcdimpodiifdjnnncaafh/data-export.html?host=myorg.sandbox.my.salesforce.com

With me this didn't work...

**Comment by forcetrekker** [comment]
Resolution: Clear force.com related cookies

From cookies in chrome browser settings, search for "*force.com" and remove all salesforce related cookies by clicking on "Remove All Shown" button.

Cause: Because salesforce changed their URL as part of latest release.

**Comment by adal877** [comment]
> Resolution: Clear force.com related cookies
> 
> From cookies in chrome browser settings, search for "*force.com" and remove all salesforce related cookies by clicking on "Remove All Shown" button.
> 
> Cause: Because salesforce changed their URL as part of latest release.

Thank you so much! It's working as a charm now <3
